mscore/albummanagerdialog.cpp

```diff
@@ -0,0 +1,55 @@
+#include "albummanagerdialog.h"
```

Comment:
Standard header missing (copyright etc.)

Reply:
Thanks, I will add it :-)